bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/transaction/actor/ImportActorMapping.java

```diff
@@ -33,7 +35,7 @@
 import org.bonitasoft.engine.identity.model.SRole;
 import org.bonitasoft.engine.identity.model.SUser;
 import org.bonitasoft.engine.persistence.SBonitaReadException;
-import org.bonitasoft.engine.xml.Parser;
+import org.xml.sax.SAXException;
```

Comment:
Should we wrap SAXException exception with bonita ones in ActorMappingConverter ?